### PR TITLE
Add new total constructed and total preserved housing units by geography

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Medium PR, should have two reviewers 🏠 
Addresses issue #291. I chose to call the new code from the wrapper function `pivot_add_total` to keep the `<geography>_hny_units_con_type` function simpler. 

The files for internal review can be un-committed after this PR is approved, no need to merge them in

### CODE REVIEW

File: aggregate/housing_production/hpd_housing_ny_affordable_housing.py [hunk 1]
```diff
@@ -91,6 +97,16 @@ def pivot_and_flatten_index(df, geography):
     return df
 
 
+def add_total(df):
+    for t in ["preservation", "newconstruction"]:
+        cols = [
+            f"units_{band}_{t}_count"
+            for band in ["eli", "vli", "li", "mi", "midi", "oi"]
+        ]
+        df[f"units_TOTAL_{t}_count"] = df[cols].sum(axis=1)
```
Comment (td928) [inline aggregate/housing_production/hpd_housing_ny_affordable_housing.py:106]: one thing to confirm here is whether the naming is units_TOTAL_{t}_count or in @AmandaDoyle  original issue #291 is _ allami_

Comment (SashaWeinstein) [reply aggregate/housing_production/hpd_housing_ny_affordable_housing.py:106]: Oh yea I don't know where I got `TOTAL` from, will re-name

**Review by td928 — CHANGES_REQUESTED** [review]
the three internal review. output files are sent to the root of housing production folder somehow. But when I run it on my machine they get sent correctly to their geography folder. Let me know if you have explanations for it @SashaWeinstein

**Comment by SashaWeinstein** [comment]
New commits should both those concerns

**Review by td928 — APPROVED** [review]
lgtm

**Pull request merged**